### DISCUSSION TITLE
Helpppp

### DISCUSSION THREAD

**Discussion by DevonYeles** [community/community#34707] [New to GitHub]
i'm trying to place this code down!!!
Authenticator Setup
(*  *)  i won't show the code...) 
View QR code

Enter the manual entry key above into your authenticator app, then enter the generated code below.

**Comment by airtower-luna** [comment]
It's not really clear what you have trouble with, but it seems like you're trying to set up TOTP for 2FA? How to enter the code (or scan it from the QR code) depends on the application you use.

**Reply by DevonYeles** [reply]
well roblox says something about using this website and want me to put a code in but idk how to put a code in and im new to this!
